db/migrate/20211206160211_create_index_on_email_tokens_token_hash.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class CreateIndexOnEmailTokensTokenHash < ActiveRecord::Migration[6.1]
```

Comment:
This one will be merged in https://github.com/discourse/discourse/pull/15202, right?

Reply:
That is correct. This migration has been already merged in another PR and this commit is building on top of that.